### PR TITLE
UAR-2184: First attempt at edge case fix where initiator is the approver in all stops.

### PR DESCRIPTION
**Pull request by nataliac06**

### CODE REVIEW

File: src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java [hunk 1]
```diff
@@ -133,6 +133,8 @@
 	private static final String EMPTY_STRING = "";
 	private static final String SUPER_USER_ACTION_REQUESTS = "superUserActionRequests";
 
+	private static final String NODE_NAME_HIERARCHY_REQUEST = "Hierarchy Request";
```
Comment (sskinner) [inline src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java]: Any reason to not use the existing constant on line148? (Or PropDevRoutingStateConstants:170?)

File: src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java [hunk 2]
```diff
@@ -332,19 +334,27 @@ else if ( variableKey.startsWith( KewApiConstants.RECEIVE_FUTURE_REQUESTS_BRANCH
...
+
+						if ( NODE_NAME_HIERARCHY_REQUEST.equalsIgnoreCase(actionRequest.getNodeName())){
+							hierarchyActionRequests++;
+							LOG.debug("MultipleApprovalRequests: actionRequestId="+actionRequest.getId()+" actionRequestAnnotation="+actionRequest.getAnnotation()+" hierarchyActionRequests="+hierarchyActionRequests);
+							if ( hierarchyActionRequests > 1)
+								return true;
+						}
+
+					} else {
+						LOG.debug("MultipleApprovalRequests: actionRequestId="+actionRequest.getId()+" actionRequestAnnotation="+actionRequest.getAnnotation()+" crtUserActiveActionRequests=1");
```
Comment (sskinner) [inline src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java]: Debug refs crtUserActiveActionRequests, but is now hierarchyActionRequests.

Comment (nataliac06) [reply src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java]: Oh indeed. I can take out that debug for good.

File: src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java (hunk 1 above)
Comment (nataliac06) [reply src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentActionsAction.java]: Didn't see that one. Ending up using a method call to make the code more readable.

**Pull request merged**